optuna/importance/__init__.py

```diff
@@ -75,6 +77,10 @@ def get_param_importances(
                 Specify this argument if ``study`` is being used for multi-objective
                 optimization. For example, to get the hyperparameter importance of the first
                 objective, use ``target=lambda t: t.values[0]`` for the target parameter.
+        normalize:
+            A boolean option to specify whether the sum of the importance values should be
```

Comment:
How about making this argument to "experimental"?

Reply:
Fixed. PTAL. @HideakiImamura